admin/check-user-guide.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$checker = new UserGuideCheck();
+$checker->run();
+
+class UserGuideCheck
+{
+    public function run(): void
+    {
+        $files = $this->getIndexedFiles();
+        $this->checkTab($files);
+    }
+
+    private function getIndexedFiles(): array
+    {
+        exec('git diff --cached --name-only --diff-filter=ACMR HEAD | grep \\.rst$', $output);
+
+        return $output;
+    }
+
+    private function checkTab(array $files): void
+    {
+        foreach ($files as $file) {
+            $contents = file_get_contents($file);
+
+            if (strpos($contents, "\t") !== false) {
+                echo "{$file}\n";
+                echo "The user guide contains tab(s). Please replace tabs with spaces before commit.\n";
+
+                exit(1);
```

Comment:
I agree that it will be better DX.
But do you really need the information? This script is for only contributors.

If there are tabs in RST, it is a simple mistake.
Then I see the file on my IDE, and replace tabs with spaces.
That is enough for me.

Reply:
I changed to exit after checking all files, instead of exiting at the first error file.